README.md

```diff
@@ -103,25 +103,95 @@ In CI linting, formatting and also run. These checks can be seen in the [CI work
 
 Visit the [Nx Documentation](https://nx.dev/getting-started/intro) to learn more.
 
-# Docker & Vegacapsule
+# 🐋 Docker
 
 ## Docker
 
-The [Dockerfile](./dockerfiles) for running the frontends is pretty basic, merely building the application with the APP arg that is passed in and serving the application from [nginx](./nginx/nginx.conf). The only complexity that exists is that there is a script which allows the passing of run time environment variables to the containers. See configuration below for how to do this.
+There are two [Dockerfiles](./dockerfiles) for running the frontends: **dist** build and **node in docker** build. For specificc build instructions follow [build instructions](#build-instructions)
+
+In order to run a container on port 3000:
+
+```bash
+docker run -p 3000:80 [TAG]
+```
+
+## Build Instructions
```

Comment:
**thought (non-blocking)**: We should rename this section. This isn't required for most people, but it is a reasonable way of setting up a hosted version for the web, or pinning a build on IPFS. 

How about `Hosting a Console`